### DISCUSSION TITLE
Link to same table records [Jan 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4828] [Discord Threads]
> 🙋‍♂️ **kantumrobot**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Yes. Link supported for records within same table & across tables  
#  
> 🙋‍♂️ **kantumrobot**  
#### <@857998597474484295>    
#### Great thanks!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1049808339414028328)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
